docs/Docker-Instructions.md

```diff
@@ -21,16 +20,47 @@ And one of:
 * CentOS 7
 * SUSE Linux Enterprise Server 12
 
-You'll need to [install the OMS Agent for Linux] (docs/OMS-Agent-for-Linux.md#steps-to-install-the-oms-agent-for-linux) on each host and then do the following to configure your containers to use the FluentD logging driver:
+## Setting up
+As a pre-requisite, docker must be running prior to this OMS Linux Agent installation. If you have installed before running docker, please re-install OMS Agent. For more information about docker, please go to https://www.docker.com/.
+
+The following settings for container host must be done to monitor the containers. 
 
- * Edit `/etc/default/docker` and add this line:
+#### Settings on container host - systemd
```

Comment:
I did add them now. Normally, docker does not even add OS and versions to these settings.